contrib/cf-diag/cf-diag.sh

```diff
@@ -53,9 +53,22 @@ trap 'printf "1..%d\n" "$test_number"' EXIT
 # I recommend custom checks be added only at the end.
 # Skip ahead to CUSTOM TESTS.
 
-# Show host key (digest)
-printf '# '
-cf-key -p /var/cfengine/ppkeys/localhost.pub 2>/dev/null || echo SHA not available
+desc="Executing user has valid ppkey"
+if ! [ $(id -u) = 0 ]; then
```

Comment:
I think using ``!=`` would be better readable here.

Reply:
Or just use `=` or better yet, `-eq`.  It's not as though it matters which path goes in "then" and which goes in "else."